query-service-impl/src/main/java/org/hypertrace/core/query/service/prometheus/PromqlQuery.java

```diff
@@ -0,0 +1,33 @@
+package org.hypertrace.core.query.service.prometheus;
+
+import java.util.List;
+
+public class PromqlQuery {
+
+  private final List<String> queries;
+  private final boolean isInstantRequest;
+  private final long startTimeMs;
+  private final long endTimeMs;
+  private final long stepMs;
+
+  public PromqlQuery(
+      List<String> queries,
+      boolean isInstantRequest,
+      long startTimeMs,
+      long endTimeMs,
+      long stepMs) {
```

Comment:
What is `stepMs` ?

Reply:
it is the timePeriod for range queries

Reply:
nit : I suppose Ms stand for millisecond. Can we use it instead ?

Reply:
this class is removed now, it is `instant` in the new class